pkgs/by-name/mo/monado/package.nix

```diff
@@ -139,17 +143,53 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   cmakeFlags = [
-    (lib.cmakeBool "XRT_FEATURE_SERVICE" serviceSupport)
+    (lib.cmakeBool "XRT_FEATURE_SERVICE" (serviceSupport && !clientLibOnly))
     (lib.cmakeBool "XRT_HAVE_TRACY" tracingSupport)
     (lib.cmakeBool "XRT_FEATURE_TRACING" tracingSupport)
     (lib.cmakeBool "XRT_OPENXR_INSTALL_ABSOLUTE_RUNTIME_PATH" true)
+  ]
+  ++ lib.optionals clientLibOnly [
+    (lib.cmakeBool "XRT_FEATURE_CLIENT_WITHOUT_SERVICE" true)
+    (lib.cmakeBool "XRT_HAVE_LIBUVC" false)
+    (lib.cmakeBool "XRT_HAVE_LIBUSB" false)
+    (lib.cmakeBool "XRT_HAVE_JPEG" false)
+    (lib.cmakeBool "XRT_HAVE_HIDAPI" false)
+    (lib.cmakeBool "XRT_HAVE_GST" false)
+    (lib.cmakeBool "XRT_HAVE_BLUETOOTH" false)
+    (lib.cmakeBool "XRT_FEATURE_DEBUG_GUI" false)
+    (lib.cmakeBool "XRT_FEATURE_WINDOW_PEEK" false)
+    (lib.cmakeBool "XRT_FEATURE_SLAM" false)
+    (lib.cmakeBool "XRT_MODULE_MONADO_CLI" false)
+    (lib.cmakeBool "XRT_MODULE_MONADO_GUI" false)
+    (lib.cmakeBool "XRT_MODULE_MERCURY_HANDTRACKING" false)
+    (lib.cmakeBool "XRT_BUILD_SAMPLES" false)
   ];
 
   # Help openxr-loader find this runtime
   setupHook = writeText "setup-hook" ''
     export XDG_CONFIG_DIRS=@out@/etc/xdg''${XDG_CONFIG_DIRS:+:''${XDG_CONFIG_DIRS}}
   '';
 
+  platformPath = # https://registry.khronos.org/OpenXR/specs/1.1/loader.html#architecture-identifiers
+    {
+      "i686-linux" = "i686";
+      "x86_64-linux" = "x86_64";
+      "aarch64-linux" = "aarch64";
```

Comment:
This limits Monado to only these systems, while `meta.platforms` is `lib.platforms.linux`. Either extend this to cover all possible platforms, or set meta.platforms to these.